src/stakingOptions.ts

```diff
@@ -238,6 +238,49 @@ export class StakingOptions {
     );
   }
 
+  /**
+   * Creates a transaction for config that includes setup instructions
```

Comment:
no need for comments like this if they arent going to provide useful information